### PR TITLE
Update cmake docs to prevent mismatching Qt version

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

**Comment by dantti** [comment]
What's the point of removing automatic selection?

**Comment by DarthGandalf** [comment]
If both Qt5 and Qt6 are installed, there's no way to compile QXlsx for one of them.

**Comment by DarthGandalf** [comment]
> Of course there is, just set the CMAKE_PREFIX_PATH

Yes, thanks. I've changed this PR to update docs instead.

File: HowToSetProject-cmake.md [hunk 1]
```diff
@@ -31,13 +32,21 @@ There are 2 possible ways:
...
 target_link_libraries(myapp PRIVATE QXlsx::QXlsx)
 ```
+
 2) Use cmake FetchContent
 In your CMakeLists.txt:
 
 ```cmake
+find_package(Qt5 ...)
+# This is to make sure QXlsx will use the same Qt version (5 vs 6) as your project. Replace with ${Qt6_DIR} if using Qt6.
+list(PREPEND CMAKE_PREFIX_PATH ${Qt5_DIR})
```
Comment (dantti) [inline HowToSetProject-cmake.md:49]: same here, see https://doc-snapshots.qt.io/qt6-dev/cmake-variable-reference.html#installation-variables

File: HowToSetProject-cmake.md [hunk 2]
```diff
@@ -31,13 +32,21 @@ There are 2 possible ways:
...
```
Comment (dantti) [inline HowToSetProject-cmake.md:37]: no need for this,  QT_DEFAULT_MAJOR_VERSION should find the right Qt version

Comment (DarthGandalf) [reply HowToSetProject-cmake.md:37]: That only affects whether the `qt_...` commands are aliases to `qt5_...` or `qt6_...` commands

**Comment by DarthGandalf** [comment]
Thanks, updated docs.

**Pull request merged**